com.unity.render-pipelines.universal/Documentation~/rendering/deferred-rendering-path.md

```diff
@@ -0,0 +1,422 @@
+
+* Bit 2, **SpecularHighlightsOff**: if set, the pixel does not receive specular highlights.
+
+* Bit 4, **SubtractiveMixedLighting**: if set, the pixel uses subtractive mixed lighting.
+
+* Bit 8, **SpecularSetup**: if set, the Material uses the specular workflow.
+
+For more technical details, see the file `/ShaderLibrary/UnityGBuffer.hlsl`.
+
+**Specular**
+
+This field contains the following values:
+
+* SimpleLit Material: RGB specular color stored in 24 bits.
+
+* Lit Material with metallic workflow: reflectivity stored in 8 bits, 16 bits are not used.
+
+* Lit Material with specular workflow: RGB specular color stored in 24 bits.
+
+**Occlusion**
+
+This field contains the baked occlusion value from the baked lighting. For real-time lighting, Unity calculates the ambient occlusion value by combining the baked occlusion value with the SSAO value.
+
+**Normal**
+
+This field contains the world space normals encoded in 24 bits. For information on the encoding of normals, see section [Encoding of normals in G-buffer](#accurate-g-buffer-normals).
+
+**Smoothness**
+
+This field stores the smoothness value for the SimpleLit and Lit materials.
+
+**Emissive/GI/Lighting**
+
+This render target contains the Material emissive output and baked lighting. Unity fills this field during the G-buffer Pass. During the deferred shading pass, Unity stores lighting results in this render target.
+
+Render target format:
+
+* **B10G11R11_UFloatPack32**, unless one of the following conditions is true:
+
+    * In URP Asset, the setting **Quality** > **HDR** is turned on, and the target Player platform does not support HDR.
+
+    * In Player Settings, the setting **PreserveFramebufferAlpha** is true.
+
+* **R16G6B16A16_SFloat**, if Unity cannot use **B10G11R11_UFloatPack32** because of the project settings.
+
+* If Unity cannot use one of the other formats in the list, it uses what the following method returns: `SystemInfo.GetGraphicsFormat(DefaultFormat.HDR)`.
+
+**ShadowMask**
+
+Unity adds this render target to the G-buffer layout when Lighting Mode is set to Subtractive or Shadow mask. 
+
+The Subtractive and the Shadow mask modes are optimized for the Forward Rendering Path, and are less efficient in the Deferred Rendering Path. In the Deferred Rendering Path,  avoid using these modes and use the Baked Indirect mode instead to improve GPU performance.
+
+**Rendering Layer Mask**
+
+Unity adds this render target to the G-buffer layout when the Light Layers feature is enabled (URP Asset, **Advanced** > **Light Layers**). The Light Layers feature might have a significant impact on the GPU performance. For more information, see section [Light Layers](#light-layers).
+
+**DepthStencil**
+
+Unity uses the four lowest bits of this render target to mark the Material type. See also [URP Pass tags: UniversalMaterialType](../urp-shaders/urp-shaderlab-pass-tags.md#universalmaterialtype). 
```

Comment:
"Unity reserves the four highest bits of this render target to mark the Material type"

To be accurate, we currently use 2 bits for material type, 1 bit for stencil and 1 bit is reserved for future use.

Reply:
Thanks!